spec/models/custom_attribute_spec.rb

```diff
@@ -10,8 +10,4 @@
   it "returns the value type of DateTime custom attributes" do
     expect(time_custom_attribute.value_type).to eq(:datetime)
   end
-
-  it "returns the value type of Fixnum custom attributes" do
-    expect(int_custom_attribute.value_type).to eq(:fixnum)
-  end
```

Comment:
Sorry, I was lazy, I didn't put the commit details (which keeps changing the commit sha1 when I rebase/force push), so I'll put it here.  Note, I totally agree with @cben here, the test removal is probably not enough.  We need to remove usage that cared if it was Fixnum vs Bignum:

CustomAttribute#value_type returns a symbol for a ruby core class, a
hardcoded value instead of the class's behavior.
Do we use this?  Why are we returning a value instead of caring about
behavior of the old Fixnum and Integer?  If we don't use this, we
should remove the whole method and it's tests added in:
ManageIQ@4687394

The PR that added this was:
ManageIQ#11000

Adding CustomAttribute to provider UI/API was added in:
ManageIQ#10897